app/helpers/data_getter.py

```diff
@@ -216,16 +216,15 @@ def get_tracks_by_event_id():
         return Track.query.filter_by(event_id=get_event_id())
 
     @staticmethod
-    def get_sessions(event_id, state='accepted'):
```

Comment:
There are a lot of duplicate functions doing similar things. We should be removing them and they will be removed during the nextgen refactor. `get_sessions` function is preferred to `get_sessions_by_state_and_event_id`.